unit/goto-programs/interval_analysis.test.cpp

```diff
@@ -392,7 +392,7 @@ TEST_CASE(
     "a = -52;\n"
```

Comment:
Actually, the comment here should read `a: [MIN_INT, -50]`. Otherwise, good catch, I just wonder why the line number change did trigger it.